skunkworks_crow/src/main/res/values/colors.xml

```diff
@@ -5,4 +5,5 @@
     <color name="colorAccent">#00796B</color>
     <color name="colorTabInactive">#CFD8DC</color>
     <color name="colorTabActive">#FFFFFF</color>
+    <color name="lightGray">#D3D3D3</color>
```

Comment:
Give some meaningful name(for e.g. if you're using it for button border then name it `buttonBorder`)

Reply:
yeah, I named it to `colorButtonDisabled`